src/ray/pubsub/python_gcs_subscriber.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 The Ray Authors.
+// Copyright 2025 The Ray Authors.
```

Comment:
Should this be under src/ray/gcs/pubsub since its about gcs pubsub?

Reply:
No -- `gcs/` should contain the implementation of the GCS. Other components subscribe, so it should be a in common directory

Reply:
We could put the `GcsPublisher` inside of `gcs/`, but I think it makes sense to keep them next to each other